_posts/2021-04-01-how-to-upload-lead-nurturing-information.md

```diff
@@ -0,0 +1,70 @@
+---
+date: 2021-04-1
+title: How to upload Lead Nurturing information
+video_id: JC2yJnBXzLM
+description: How to upload lead nurturing information to the assistant so that all the prospects are nurtured.
+type: How-to
+# categories:
+#   - assigning-leads
+# set: getting-started
+# set_order: 3
+---
+One of the most important features of 7Targets is lead nurturing. You can upload nurturing messages to each assistant in just a few simple steps. AI Sales assistant then dynamically attaches these nurturing messages to every follow-up email in the 'PS' section.
+
+Follow these simple steps to upload the nurturing information.
+
+## Step 1
+
+Sign-in into your account.
+
+Click on 'Assistant' as shown below.
+
+![image](../images/lead-nurturing-1.png)
+
+You'll be directed to this page.
+
+![image](../images/lead-nurturing-2.png)
+
+
+## Step 2
+
+Select the assistant you want to add nurturing information to.
+
+Click on the 'Nurture' button of that assistant as shown.
+
+![image](../images/lead-nurturing-3.png)
+
+## Step 3
+
+Click on the 'Add Row' button to add the nurturing message.
+
+![image](../images/lead-nurturing-4.png)
+
+## Step 4
+
+Now either you can edit the message in the section shown, or you can paste the already available message here.
+
+![image](../images/lead-nurturing-5.png)
+
+This is a sample of how your nurturing messages can be. You should copy the messages one at a time.
+
+![image](../images/lead-nurturing-6.png)
+
+## Step 5
+
+Paste it in the section shown below.
+
+To add the next message, click on the '+' (plus) button and then repeat step 4.
+
+
+![image](../images/lead-nurturing-7.png)
+
+## Step 6
+
+Click on 'Save' button at the left bottom corner. 
+
+![image](../images/lead-nurturing-8.png)
+
+Since you can assign different assistants for different verticals, you may want to upload vertical-specific nurturing messages for the respective assistant.
```

Comment:
Change this to below.

We recommend, you have vertical/industry specific Assistant. Success story or testimonial from other customer of same vertical/industry would interest leads better.